.github/workflows/build-pdf.yml

```diff
@@ -28,15 +28,11 @@ on:
         required: false
         type: boolean
         default: false
-      pages:
-        description: Deploy HTML to Github pages?
-        required: false
-        type: boolean
-        default: false
   pull_request:
   push:
     branches:
       - main
+      - automatic-pages-deployment
```

Comment:
Will remove these changes before merging, just couldn't figure out another way to test the CI actually works.